sdk/provisioning/Azure.Provisioning/tests/Infrastructure/WebSiteUsingL3/resources/rg_TEST_module/rg_TEST_module.bicep

```diff
@@ -145,7 +145,6 @@ resource sqlServer_RgHXTrwDE 'Microsoft.Sql/servers@2022-08-01-preview' = {
 resource sqlDatabase_6M6mjEKjO 'Microsoft.Sql/servers/databases@2022-08-01-preview' = {
   parent: sqlServer_RgHXTrwDE
   name: 'db-TEST'
-  location: 'westus'
```

Comment:
I think I just forgot to re-run these last time. This is correct now - location shouldn't be included for child resources. We should probably set up the CI to regenerate these and do a diff.